src/components/course/data/utils.jsx

```diff
@@ -373,8 +380,7 @@ export function shouldUpgradeUserEnrollment({
 
 // Truncate a string to less than the maxLength characters without cutting the last word and append suffix at the end
 export function shortenString(str, maxLength, suffix, separator = ' ') {
-  if (str.length <= maxLength) { return str; }
-  return `${str.substr(0, str.lastIndexOf(separator, maxLength))}${suffix}`;
+  return str.length <= maxLength ? str : `${str.substring(0, str.lastIndexOf(separator, maxLength))}${suffix}`;
```

Comment:
[curious] Why was this refactored? As far as I can tell, the logic is the same but it's now a bit more difficult to understand than having the explicit `if` conditional and early return, without much added benefit.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr

IDE was giving a deprecation warning of `.substr` method and I thought I would 1 liner it in the process. I can refactor back to the if statement 👍🏽  but with the non deprecated `.substring` method 👍🏽